__tests__/integration/repeatableResources.test.js

```diff
@@ -6,28 +6,40 @@ import { testUserLogin } from './loginHelper'
 describe('Adding new embedded Resource Templates', () => {
   beforeAll(async () => {
     await testUserLogin()
-  })
-
-  it('loads up a resource template from the list of loaded templates', async () => {
```

Comment:
I'm not a fan of "tests" that are really about setup -- so I moved these lines to beforeAll.